ui/tests/integration/components/recovery/page/snapshots-test.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { dateFormat } from 'core/helpers/date-format';
+
+const SELECTORS = {
+  badge: (name) => `[data-test-badge="${name}"]`,
+};
+
+module('Integration | Component | recovery/snapshots', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.model = {
+      snapshots: [],
+      canLoadSnapshot: false,
+    };
+    this.version = this.owner.lookup('service:version');
+    this.version.type = 'enterprise';
+  });
+
+  test('it displays empty state in CE', async function (assert) {
+    this.version.type = 'community';
+    await render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);
```

Comment:
In the future, you can always make this a convenience function in the `beforeEach` hook so if the component args ever change, we only have to update it in one place rather than each test
```js
    this.renderComponent = () => render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);
    // in a test
    await this.renderComponent()
```